src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -50,6 +50,7 @@ public sealed class VsSolutionRestoreService : IVsSolutionRestoreService, IVsSol
         private const string RestoreSources = nameof(RestoreSources);
         private const string RestoreFallbackFolders = nameof(RestoreFallbackFolders);
         private const string AssetTargetFallback = nameof(AssetTargetFallback);
+        private const string RestoreAdditionalProjectFallbackFoldersExcludes = nameof(RestoreAdditionalProjectFallbackFoldersExcludes);
         private const string RestoreAdditionalProjectFallbackFolders = nameof(RestoreAdditionalProjectFallbackFolders);
         private const string RestoreAdditionalProjectSources = nameof(RestoreAdditionalProjectSources);
```

Comment:
does it make sense to move this to a comman class and declare them as public static since these are being used in multiple classes?